input/kafkamdm/kafkamdm.go

```diff
@@ -262,19 +300,16 @@ func (k *KafkaMdm) consumePartition(topic string, partition int32, currentOffset
 				log.Error(3, "kafka-mdm failed to commit offset for %s:%d, %s", topic, partition, err)
 			}
 			k.lagMonitor.StoreOffset(partition, currentOffset, ts)
-			offset, err := k.client.GetOffset(topic, partition, sarama.OffsetNewest)
+			newest, err := k.client.GetOffset(topic, partition, sarama.OffsetNewest)
```

Comment:
for consistancy I think this should use `currentOffset, err = k.tryGetOffset(topic, partition, sarama.OffsetOldest, 1, time.Second)`

Reply:
if you mean "use tryGetOffset", that's fine I can do that
but assigning the oldest offset to currentOffset doesn't make sense here. the purpose of this code is update `newest` to be the currently newest offset.

Reply:
i meant use `k.tryGetOffset` instead of calling `k.client.GetOffset` directly.